Project_0.Altis/MessageLoop/MessageLoop.sqf

```diff
@@ -31,7 +31,7 @@ CLASS("MessageLoop", "");
 	//Mutex for accessing the message queue
 	VARIABLE("mutex");
 	//Debug name to help read debug printouts
-	VARIABLE("debugName");
+	VARIABLE("name");
```

Comment:
Did you mass replace debugName in all our files and did not check it? I don't think that we wanted to rename this in MessageLoop? Or did we?

Reply:
If you did not intend to do it in this class, can you please be more careful and not do ~~gen~~ mass replacements in all files?
Probably here I don't care if it's setName or setDebugName in MessageLoop.

Reply:
i saw it and thought that it was ok since nothing was broken and debugName has no meaning since there is no name prop already present

Reply:
Cool, I think I wanted to use debugName for msgLoop here:
https://github.com/Sparker95/Project_0/blob/a6c0ec99e6ede91b8f448f3720df8a14f4dcefc0/Project_0.Altis/MessageLoop/fn_threadFunc.sqf#L19-L22
but forgot to do it or gave it up